dash-info.yaml

```diff
@@ -1,10 +1,2612 @@
+                htmlBig(children="This text may or may not appear slightly larger.")
+               )
+              )
+            )
+
+            app$run_server()
+    - name: htmlBlink
+      dontrun: TRUE
+      code: |
+            # The blink tag is now obsolete and deprecated.
+            # It may not function properly in all browsers,
+            # and it may cease working without warning.
+            #
+            # This element is generally unsupported on all
+            # modern browser releases.
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlP(children="Here is a bit of text."),
+                htmlBlink(children="Here is a bit of blinking text.")
+               )
+              )
+            )
+
+            app$run_server()
+    - name: htmlBlockquote
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlP("Here is some text."),
+                htmlBlockquote(children=list(
+                   htmlP("And here is a quotation in block format.")
+                   )
+                )
+               )
+              )
+            )
+
+            app$run_server()
+    - name: htmlBr
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlP("Here is some text."),
+                htmlBr(),
+                htmlP("Here is additional text."),
+                htmlBr(),
+                htmlP("See the gap in between the lines?")
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlButton
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlButton("Click me!")
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlCanvas
+      dontrun: TRUE
+      code: |
+            # this component requires JavaScript code
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlCanvas(id="canvas-component")
+                )
+              )
+            )
+
+            app$run_server()
+    - name: htmlCaption
+      dontrun: TRUE
+      code: |
+            library(dash)
+            library(dashHtmlComponents)
+
+            app <- Dash$new()
+
+            app$layout(
+              htmlDiv(list(
+                htmlCaption("This is an example of htmlCaption.")
```

Comment:
intended to go in a `<table>` https://www.w3docs.com/learn-html/html-caption-tag.html

Reply:
fixed in f462bce